docs/CAVEATS.md

```diff
@@ -0,0 +1,17 @@
+# Caveats
+## Only iceberg V2 table supported
+This connector only writes using iceberg table V2 spec (no rewrite for delete and upsert - merge on read only)
+
+## Only hadoop catalog supported
+This connector only packages with support for `hadoop` catalog.
+
+## No automatic schema evolution
+Currently, there is no handler to detect schema changes and auto evolve the schema. Schema change events can make the connector throw error. To workaround this, turn off schema change event in `source` setting.
+
+- For SQL Server, set `debezium.source.include.schema.changes=false`
+
+## Specific tables replication
+By default, debezium connector will publish all snapshot of the tables in the database, that leads to unnessesary iceberg table snapshot of all tables. Unless you want to replicate all table from the database into iceberg table, set `debezium.source.table.include.list` to specific tables that you want to replicate. By this way, you avoid replicate too many table that you don't really want to.
+
+## AWS S3 credentials
+You should inject environment variables `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY` to write to S3 or setup proper `HADOOP_HOME` env then add s3a configuration into `core-site.xml`, more information can be found [here](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html#Authenticating_with_S3).
```

Comment:
i believe its is possible to use `application.properties` to configure AWS S3
all the settings starting with `debezium.sink.iceberg.<my.config>` are passed to iceberg and from there used by iceberg

example from unit test
https://github.com/memiiso/debezium-server-iceberg/blob/master/debezium-server-iceberg-sink/src/test/java/io/debezium/server/iceberg/testresources/S3Minio.java#L131

Reply:
Yup, saw that too, by default, I suggest turn off `debezium.sink.iceberg.fs.s3a.aws.credentials.provider=com.amazonaws.auth.DefaultAWSCredentialsProviderChain` to provide easier way to inject aws credential